docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eksctl.md

```diff
@@ -0,0 +1,288 @@
+
+```shell
+ eksctl create iamidentitymapping --cluster  <clusterName> --region=<region> --arn arn:aws:iam::123456:role/testing --group system:masters --username admin
+```
+
+Where `arn` is the `arn` of your user or the role. The `group` is the Kubernetes rule, where `system:masters` is equivalent to an admin role. Lastly `username` is either the username itself or the role name, which is used for logs.
+
+More information about usage and other configuration options can be found in the [eksctl documentation](https://eksctl.io/usage/iam-identity-mappings/).
+
+#### kubectl
+
+The same can also be achieved by using `kubectl` and manually adding the mapping as part of the `mapRoles` or `mapUsers` section.
+
+```shell
+kubectl edit configmap aws-auth -n kube-system
+```
+
+For detailed examples, check out the [documentation](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html) provided by AWS.
+
+## Postgres Database
+
+Creating a Postgres Database can be solved in various ways, one of those utilizing the UI or the AWS CLI.
+For the purposes of this guide, we're trying to provide you with a reproducible setup, therefore aiming for the CLI. Using the UI may be easier and can be followed by [conducting the AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_GettingStarted.CreatingConnecting.PostgreSQL.html).
+
+1. Figure out the VPC that is associated with the cluster
+   - Either via the [AWS UI](https://console.aws.amazon.com/eks/home)
+   - Or CLI - `aws ec2 describe-vpcs` and look for the VPC that contains the tag
+   ```yaml
+   {
+       "Key": "alpha.eksctl.io/cluster-name",
+       "Value": "<clusterName>"
+   },
+   ```
+2. Copy the `vpcId` to use for the next step
+3. Create security group within the VPC for the aurora postgres connection
+
+```shell
+aws ec2 create-security-group \
+  --group-name aurora-postgres-sg \
+  --description "Security Group to allow the k8s cluster to connect to aurora postgres" \
+  --vpc-id <vpcId>
+```
+
+4. Copy the resulting output of the `groupId`
+5. Create security ingress rule to allow access to postgres
+
+```shell
+aws ec2 authorize-security-group-ingress \
+  --group-id <groupId \
+  --protocol tcp \
+  --port 5432 \
+  --cidr 10.192.0.0/16
+```
+
+5. Create db subnet group to associate RDS with within the existing VPC
+
+```shell
+aws rds create-db-subnet-group \
+    --db-subnet-group-name camunda-postgres \
+    --db-subnet-group-description "Subnet for Camunda Postgres" \
+    --subnet-ids subnet-0659823f3a8fa383c subnet-0f435ed77e62a13be subnet-06de5f9aa67849b1c
+```
+
+6. Create RDS cluster within private subnet of the VPC. More configuration options can be found in the [AWS documentation](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/create-db-cluster.html).
+
+```shell
+aws rds create-db-cluster \
+    --db-cluster-identifier <dbCluster> \
+    --engine aurora-postgresql \
+    --engine-version 15.2 \
```

Comment:
you mean just `15.x` or `15.4`?
It will eventually be outdated anyway. I'll add a link to the supported environments https://docs.camunda.io/docs/reference/supported-environments/#camunda-8-self-managed